templates/vue/src/components/TapestryMain/TapestryNode/index.vue

```diff
@@ -282,6 +284,12 @@ export default {
         .filter(n => n.status !== "draft")
       return rows.filter(row => row.completed).length / rows.length
     },
+    highlightNode() {
+      return (
+        this.node.reviewStatus !== nodeStatus.ACCEPT ||
+        this.settings.showAcceptedHighlight
```

Comment:
should this be `this.node.reviewStatus == nodeStatus.ACCEPT && this.settings.showAcceptedHighlight`

Reply:
@jaskwong No, we want to always highlight the non-accepted thus the `this.node.reviewStatus !== nodeStatus.ACCEPT`. Accepted node highlighting is determined by the `this.settings.showAcceptedHighlight`.